addons/mail/models/mail_channel.py

```diff
@@ -263,9 +263,11 @@ def write(self, vals):
         if 'image_128' in vals:
             notifications = []
             for channel in self:
-                notifications.append([channel, 'mail.channel/insert', {
-                    'avatarCacheKey': channel._get_avatar_cache_key(),
-                    'id': channel.id,
+                notifications.append([channel, 'record/insert', {
+                    'Channel': [{
```

Comment:
the extra array look useless in all of those cases, insert in JS should work both for single or multi data